src/models/CommonModel.ts

```diff
@@ -147,11 +164,7 @@ export class CommonModel extends CommonSchema<CommonModel> {
         mergeTo.properties = mergeFromProperties;
       } else {
         for (const [propName, prop] of Object.entries(mergeFromProperties)) {
```

Comment:
I do not like that you are inconsistent. Sometimes you use `forEach` sometimes `for of` imo you should stick to `forEach`.
Also I think that you should always pass an array of properties into .addProperty method. It would be more DRY.

Reply:
added an issue for this: https://github.com/asyncapi/generator-model-sdk/issues/192